feat(draft3): add support for draft3 (#270)
sourcemeta/learnjsonschema.com: content/draft3/core/divisibleBy.markdown

---
keyword: "divisbleBy"
signature: "Number"

> value: This keyword must be set to a number that is greater than zero

Reviewer: According the spec, the number cannot be zero, but seems negative numbers are allowed?

Author: I suppose, technically negatives are allowed..

```suggestion
value: This keyword must be set to a number that is not equal to zero
```
